src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -96,8 +101,8 @@ class TerminalApp::GlobalAppSettings final
     std::wstring _wordDelimiters;
     bool _copyOnSelect;
     winrt::Windows::UI::Xaml::ElementTheme _requestedTheme;
-
     winrt::TerminalApp::LaunchMode _launchMode;
+    std::optional<std::wstring> _activePaneBorderColor{ std::nullopt };

```

Comment:
None of the rest of these private variables are initialized here. Are they initialized in the constructor instead?